detect_secrets/core/audit.py

```diff
@@ -173,6 +175,9 @@ def _secret_generator(baseline):
     )
 
     for filename, secrets in baseline['results'].items():
+        if not os.path.exists(filename):
+            print('{} does not exist.'.format(filename))
```

Comment:
I'm debating the use of this case.

1. `print` statements shouldn't be in the `_secret_generator`.
2. If the file is gone, there's nothing to audit (and we should probably just delete its entry in the updated baseline). After all, we already exclude it from the count (from your change above).

Reply:
I was ambivalent too, wasn't sure if we wanted `audit` to only have `is_secret` diffs, but yeah, removing all dead files in `audit_baseline` before calling `_secret_generator` etc. is cleaner, I'll change.